items.json

```diff
@@ -13486,6 +13486,44 @@
       }
     ]
   },
+  {
+    "id": "crpg_Heavy_Highland_Tower_Shield",
+    "name": "Heavy Highland Tower Shield",
+    "culture": "Battania",
+    "type": "Shield",
+    "price": 9802,
+    "weight": 12.0,
+    "tier": 10.335989,
+    "requirement": 0,
+    "flags": [
+      "ForceAttachOffHandPrimaryItemBone",
+      "WoodenParry",
+      "HeldInOffHand"
+    ],
+    "weapons": [
+      {
+        "class": "LargeShield",
+        "itemUsage": "hand_shield",
+        "accuracy": 100,
+        "missileSpeed": 0,
+        "stackAmount": 125,
+        "length": 140,
+        "balance": 0.0,
+        "handling": 55,
+        "bodyArmor": 10,
+        "flags": [
+          "HasHitPoints",
+          "CanBlockRanged"
+        ],
+        "thrustDamage": 5,
+        "thrustDamageType": "Blunt",
+        "thrustSpeed": 55,
+        "swingDamage": 0,
+        "swingDamageType": "Blunt",
+        "swingSpeed": 55
+      }

```

Comment:
Seems like Length does nothing , see with meow

Reply:
the shield has a physically larger model i believe that's why Diggles wanted to add it